src/main/kotlin/no/nav/syfo/domain/Personident.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.syfo.domain
+
+data class Personident(val value: String) {
+    private val elevenDigits = Regex("^\\d{11}\$")
+
+    init {
+        if (!elevenDigits.matches(value)) {
+            throw IllegalArgumentException("Value is not a valid PersonIdentNumber")
+        }
+    }
+}
```

Comment:
Kanskje vi kunne begynt å bruke inline value classes for disse? 
https://kotlinlang.org/docs/inline-classes.html

Vetle introduserte meg for det tidligere i sommer 👌🏼 Brukes i `isbehandlerdialog`: https://github.com/navikt/isbehandlerdialog/pull/95/files#r1262669921